core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -162,7 +162,7 @@ private SqlLibraryOperators() {
    * <p>MySQL has "DATEDIFF(date, date2)" and "TIMEDIFF(time, time2)" functions
    * but Calcite does not implement these because they have no "timeUnit"
    * argument. */
-  @LibraryOperator(libraries = {MSSQL, REDSHIFT, SNOWFLAKE})
+  @LibraryOperator(libraries = {MSSQL, REDSHIFT, SNOWFLAKE, HIVE})
```

Comment:
testDateDiff() and testDateSub()

Reply:
date_diff is not compative，and had test for date_sub and repeat function @ILuffZhe